apps/load-generator/main.py

```diff
@@ -112,14 +144,21 @@ def main():
         Scaler(config["scaler"], url, req_kwargs).run()
         return
 
-    hist = Histogram("loadgen_query_duration_seconds", "Query duration", 
-            ["prometheus", "group", "query"],
-            buckets=(0.01, 0.05, 0.1, 0.3, 0.7, 1.5, 3, 6, 12, 18, 28))
+    queryTimeHist = Histogram("loadgen_query_duration_seconds", "Query duration",
+        ["prometheus", "group", "queryString", "queryType", "start", "end", "step"],
+        buckets=(0.01, 0.05, 0.1, 0.3, 0.7, 1.5, 3, 6, 12, 18, 28))
```

Comment:
Adding too many buckets its expensive but as query time has such a large variance, let's extend this even further and have more precision in the upper buckets. The lower ones can be reduced somewhat, because we don't care all that much about well-performing ones.

Actually, let's just split our metrics into range and instant queries explicitly:

* `loagen_query_range_duration_seconds` with buckets `(0.01, 0.1, 0.3, 0.8, 1.2, 2.5, 5, 9, 15)`
* `loadgen_query_instant_duration_seconds` with buckets `(0.5, 1, 2.5, 4, 8, 12, 16, 20, 25, 30, 36, 45)`